### ISSUE TITLE
Breaks ActiveRecord::Associations::CollectionProxy objects within Pry rails console

### ISSUE THREAD

**Issue by 0duaht** [schneems/attendance#1]
TLDR: To duplicate the issue, start a console session using [Pry](https://github.com/rweng/pry-rails) as your default Rails console. Then try calling `.empty?` on an `ActiveRecord::Associations::CollectionProxy` object. You'll have to assign the object to a value, and then try calling `.empty?` on that value.

**Possible Explanation:**
Both `ActiveRecord::Associations::CollectionProxy` and `ActiveRecord::Relation` have different definitions of the `empty?` method, even though the former subclasses the latter.

`empty?` definition in `ActiveRecord::Associations::CollectionProxy`:
```ruby
Owner: ActiveRecord::Associations::CollectionProxy
Visibility: public
Number of lines: 3

def empty?
  @association.empty?
end
```

`empty?` definition in `ActiveRecord::Relation`:
```ruby
Owner: ActiveRecord::Relation
Visibility: public
Number of lines: 4

def empty?
  return @records.empty? if loaded?
  !exists?
end
```

Without Pry, in the default rails console, the alias call works correctly by aliasing `blank?` on both objects to the respective definitions of `empty?` in each class.

However, within the Pry console, the alias functionality breaks. `empty?` in both objects point to the definition in `ActiveRecord::Relation`, thus raising this error for `ActiveRecord::Associations::CollectionProxy` objects:
```ruby
NoMethodError: undefined method `empty?' for nil:NilClass
```
